server/app/com/xsn/explorer/data/anorm/dao/StatisticsPostgresDAO.scala

```diff
@@ -61,6 +63,34 @@ class StatisticsPostgresDAO {
       )
       .as(parseSummary.single)
   }
+
+  def getCoinsInAddresses(addresses: List[String])(implicit conn: Connection): BigDecimal = {
+    if(addresses.length == 0)
+      0
+    else {
+      val addressesList = "'" + addresses.mkString("','") + "'"
```

Comment:
This allows sql inyection because a string can be anything, in any case, it seems like we can remove this method